tools/packer/etc/motd

```diff
@@ -52,6 +52,13 @@ You only need to do this once, after which, you can enable and start the miner.
 
 
 
+Do you want to access your lotus daemon remotely? Learn how to setup token authentication
+and use client libraries from lotus docs.
+
+https://docs.filecoin.io/build/lotus/enable-remote-api-access/
```

Comment:
Note: we should add some warnings about the risks of exposing lotus API to the internet to this doc